components/layout/main.jsx

```diff
@@ -0,0 +1,12 @@
+import React from 'react'
+import { classNames } from '@oacore/design/lib/utils'
+
+import styles from './styles.css'
+
+const Main = ({ children, className, tag: Tag = 'main', ...restProps }) => (
```

Comment:
You kind of like to pass this `tag: Tag = '...'` everywhere. I don't know if it is a good idea. I see meaning in some cases. But for example, here it's completely useless. This component is used only once and I doubt it will be reused.

Reply:
It is more about consistency and clear implicit API. I prefer to allow passing rest props to actual DOM and changing node name. This is mostly for implementing accessibility too.

Reply:
Also, I am considering reuse the element in 'activities' or actual pages to avoid using fragments.

Reply:
> This is mostly for implementing accessibility too.

I don't agree. If we do it like this everywhere the a11y eslint warnings/errors will never occur. Unfortunately, eslint is not that smart and cannot detect which element is representing Tag.

Reply:
Good point 🤔